src/main.py

```
@@ -154,7 +155,8 @@ def analyze_route():
             result = analyze_image(filepath, unique_filename)
             return jsonify(result)
         except Exception as e:
```

Pylint (4.0.4)

[warning] Catching too general exception Exception

(W0718)